### ISSUE TITLE
Color codes printed when stderr is not a console

### ISSUE THREAD

**Issue by lenticularis39** [viktormalik/diffkemp#197]
SimpLL always prints color codes, even if stderr is not a console, which leads to them being written in a log file when redirecting stderr:

```
  } ^[[1;32mequal
^[[0m  Comparing "skb_cloned" and "skb_cloned" {
    Comparing "atomic_read" and "atomic_read" {
    } ^[[1;32mequal
^[[0m  } ^[[1;32mequal
^[[0m  Comparing "skb_set_tail_pointer" and "skb_set_tail_pointer" {
    Comparing "skb_reset_tail_pointer" and "skb_reset_tail_pointer" {
    } ^[[1;32mequal
^[[0m  } ^[[1;32mequal
^[[0m  Comparing "skb_shared" and "skb_shared" {
  } ^[[1;32mequal
^[[0m  Comparing "skb_drop_list" and "skb_drop_list" {
```

Ideally there would be a check in the Color:: functions to omit the colors when stderr is not a console. (This would get more complicated if we also decide to use colors when printing the results, since the conversion functions would have to know whether the output is for stdout or stdin.)

**Comment by Kuba314** [comment]
Should I include the check just for stderr? I could also check if BOTH `stdout` and `stderr` are terminals and only then print the escape sequences.

**Comment by Kuba314** [comment]
I just noticed that there are some irregularities when printing final result.
If both functions are equal, it gets printed twice. Once from simpll (`ModuleAnalysis.cpp:200`) and once from python wrapper (`semdiff/function_diff.py:301`) if `verbosity > 0`. Also, I've noticed that I forgot to use red color when printing out that "functions are syntactically different`. Also, should there be the word `syntactically`? They're equal *semantically*, right?
![image](https://user-images.githubusercontent.com/35199722/138260997-122c1c2c-85fe-4216-ac4d-9e330056e89f.png)
![image](https://user-images.githubusercontent.com/35199722/138261035-439cc216-710f-4399-928d-4131b1156250.png)
